### PR TITLE
KOGITO-4421 unify empty state across management console

### PR DESCRIPTION
**Pull request by AjayJagan**
Description : This pr unifies the empty states across the management-console pages namely : Process list, domain explorer and jobs.
JIRA: https://issues.redhat.com/browse/KOGITO-4421

### CODE REVIEW

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/DomainExplorerTable.tsx [hunk 1]
```diff
@@ -380,28 +366,65 @@ const DomainExplorerTable: React.FC<IOwnProps & OUIAProps> = ({
...
+    rows = [
+      {
+        rowKey: '1',
+        cells: [
+          {
+            props: { colSpan: 8 },
+            title: (
+              <KogitoEmptyState
+                type={KogitoEmptyStateType.Search}
+                title="No data available"
```
Comment (nmirasch) [inline ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/DomainExplorerTable.tsx]: @AjayJagan We need to  unify messages across management console, Maybe makes sense to have two components to encapsutale this KogitoNoResultsFound and KogitoNoFilterSelected,( under your criteria). At least, here the messages should be aligned with jobs and process instances:  'No results found' + 'Try using different filters'

**Review by nmirasch — COMMENTED** [review]
@AjayJagan Please could you unify empty state messages for no filter applied across Management Console
image

I see differences between the No results found.. the domain explorer header shows sort controls and  different messages
image

**Comment by Sara4994** [comment]
+1 for @nmirasch 's suggestions, also noticed columns names on Domain Explorer table moves towards left while displaying empty state.

**Comment by AjayJagan** [comment]
Thanks @Sara4994  for the help :+1: . The column space issue while displaying empty state is resolved.
domain-explorer-emptystate. 
process-list-emptystate

@nmirasch , the messages in domain explorer empty state and process list have been changed accordingly.The sorting controls have been removed while displaying the empty state. please have a look

**Review by nmirasch — APPROVED** [review]

**Review by Sara4994 — APPROVED** [review]

**Review by jstastny-cz — APPROVED** [review]
LGTM

**Pull request merged**